CIP-0015/test-vector.md

```diff
@@ -56,10 +57,11 @@ a3d63f26cd94002443bc24f24b0a150f2c7996cd3a3fd247248de396faea6a5f
     '1': '0x0036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0',
```

Comment:
Probably we should have a test case for both the old value of `1` and a new test for the new value

Reply:
Yes, if this is fully backwards compatible.  Otherwise you would presumably have one test for the original format and a different test for the new format